src/styles/main.scss

```
@@ -0,0 +1,57 @@
+@import 'variables';
+
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 $pd;
+  gap: $gp;
+  width: 726px;
+  &__day {
```

```suggestion
  width: 726px;
  
  &__day {
```
- Add a space between code blocks
- Don't hardcode width, you have width of a cell in a variable, and gaps. Calculate it